lib/lita/handlers/totems.rb

```diff
@@ -267,6 +302,19 @@ def queued_by_user(user_id)
         end
       end
 
+      def send_stats_to_signalFX(metric, value)
+        signalfx_client.send( 
+          gauges:
+           [ 
```

Comment:
Do we need to be defensive at all for this call? Should it be wrapped in a try/catch block so that it doesn't inadvertently cause issues with totem handling?

Reply:
When testing it locally and on staging, if I had an error (401) nothing broke (totems wise) but the data didnt reach signalFX, I can add it if you feel it can be safer